include/deal.II/grid/grid_generator.h

```diff
@@ -501,7 +504,14 @@ namespace GridGenerator
    * point will be the sum of these two vectors.  Colorizing is done in the
    * same way as in hyper_rectangle().
    *
-   * @note This function is implemented in 2d only.
+   * @note This function is implemented in 2d only. 
+   * 
+   * @param tria Triangulation to be created.
+   * @param corners Adjacent points. The @p dim
+   * adjacent points are the ones given in the second argument and the fourth
+   * point will be the sum of these two vectors.
```

Comment:
We should combine this text with the paragraph above (or simply delete that paragraph): there is no need to say the same thing twice.